### DISCUSSION TITLE
Is it possible to connect Android to local AGH via DoT?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#4840] [Q&A]
Android supports DoT. It allows user to input DoT domain name like dns.adguard.com. Initial resolution of DoT domain (dns.adguard.com) is performed via plaintext UDP port 53 using whichever WLAN DNS IP or carrier-assigned DNS IP. Is it actually possible to connect Android to local AGH server if AGH is not exposed to WAN? Would it work with self-signed DoT certificate?

**Comment by MkQtS** [comment]
Yes. You can install AdGuardHome on your router (with supported system, like OpenWrt), then deploy a domain with certificate to the LAN ip. Android supports dot using 853 port, just enable it in AdGuardHome.
